Makefile

```diff
@@ -81,17 +81,13 @@ vet:
 install-tools:
 	go install golang.org/x/lint/golint
 
-.PHONY: agent
-agent:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/ocagent_$(GOOS) $(BUILD_INFO) ./cmd/ocagent
-
 .PHONY: collector
 collector:
 	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/occollector_$(GOOS) $(BUILD_INFO) ./cmd/occollector
 
-.PHONY: unisvc
-unisvc:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/unisvc_$(GOOS) $(BUILD_INFO) ./cmd/unisvc
+.PHONY: otelsvc
```

Comment:
let's call it otsvc

Reply:
@SergeyKanzhelev looks like the community is split on what prefix is best to use (see here in gitter https://gitter.im/open-telemetry/community?at=5d099d6f325b465cc3609206).

I think we keep this as is for now. Once we have an agreement on what prefix to use for OpenTelemetry it is easy to rename this.